app/models/installed_heat_source.rb

```diff
@@ -0,0 +1,18 @@
+class InstalledHeatSource
+  include Virtus.model
+
+  attribute :key, String
+  attribute :name, String
+  attribute :units, Float
+  attribute :heat_production, Float
```

Comment:
I'm curious about the names `heat_production` and `heat_capacity`. Are these the heat technology versions of `demand` and `capacity`? If so, what motivated the difference in naming? Is there a limitation in the ETEngine importer such that we have to use different names?